src/main/java/duke/task/Todos.java

```diff
@@ -0,0 +1,52 @@
+package duke.task;
+
+import java.util.Date;
+
+public class Todos implements Task {
+
+    private boolean done = false;
+    private String name;
+
+    public Todos(String name) {
+        this.name = name;
+    }
+    @Override
+    public boolean isDone() {
+        return done;
+    }
+
+    @Override
+    public String getTaskName() {
+        return name;
+    }
+
+    @Override
+    public String getTaskType() {
+        return "T";
+    }
+
+    @Override
+    public String getDoneString() {
```

Comment:
May I suggest doing this method in Task, as an abstract class, as this will be very much reused and it is troublesome to edit if you ever decide to change icons/add functionality (we'd like to avoid regression).